libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -125,61 +125,81 @@ const AP_Param::GroupInfo AP_MotorsHeli_RSC::var_info[] = {
+    // @Range: 0 40
     // @Units: %
     // @Increment: 1
     // @User: Standard
-    AP_GROUPINFO("GOV_DISGAG", 14, AP_MotorsHeli_RSC, _governor_disengage, AP_MOTORS_HELI_RSC_GOVERNOR_DISENGAGE_DEFAULT),
+    AP_GROUPINFO("AROT_PCT", 18, AP_MotorsHeli_RSC, _ext_gov_arot_pct, 0),
+#endif
+
+    // @Param: CLDWN_TIME
+    // @DisplayName: Cooldown Time
+    // @Description: Will provide a fast idle for engine cooldown by raising the Ground Idle speed setting by 50% for the number of seconds the timer is set for. A setting of zero disables the fast idle. This feature will only apply after the runup complete has been declared. At any time during fast idle, disarming will shut the engine down however the timer to declare below critical speed (which triggers ground idle state) will be extended by cooldown time to keep premature engine shutdown after autonomous landing.
+    // @Range: 0 120
+    // @Units: s
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("CLDWN_TIME", 19, AP_MotorsHeli_RSC, _cooldown_time, 0),
+
+    // @Param: GOV_COMP
+    // @DisplayName: Governor Torque Compensator
+    // @Description: Adjusts the autothrottle governor torque compensator that determines how fast the governor will adjust the base torque reference to compensate for changes in density altitude. If RPM is low or high by more than 2-5 RPM, increase this setting by 1% at a time until the governor speed matches your RPM setting. Setting the compensator too high can result in surging and throttle "hunting". Do not make large adjustments at one time
+    // @Range: 0 70
+    // @Units: %
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("GOV_COMP", 20, AP_MotorsHeli_RSC, _governor_compensator, 25),
 
     // @Param: GOV_DROOP
-    // @DisplayName: Governor Droop Response
-    // @Description: Governor droop response under load, normal settings of 0-100%. Higher value is quicker response but may cause surging. Setting to zero disables the governor. Adjust this to be as aggressive as possible without getting surging or over-run on headspeed when the governor engages. Setting over 100% is allowable for some two-stage turbine engines to provide scheduling of the gas generator for proper torque response of the N2 spool
-    // @Range: 0 150
+    // @DisplayName: Governor Droop Compensator
+    // @Description: AutoThrottle governor droop response under load, normal settings of 0-50%. Higher value is quicker response to large speed changes due to load but may cause surging. Adjust this to be as aggressive as possible without getting surging or RPM over-run when the governor responds to large load changes on the rotor system
+    // @Range: 0 100
     // @Units: %
     // @Increment: 1
     // @User: Standard
-    AP_GROUPINFO("GOV_DROOP", 15, AP_MotorsHeli_RSC, _governor_droop_response, AP_MOTORS_HELI_RSC_GOVERNOR_DROOP_DEFAULT),
+    AP_GROUPINFO("GOV_DROOP", 21, AP_MotorsHeli_RSC, _governor_droop_response, 25),
```

Comment:
I believe Chris did this because the new use of this parameter changed and there wasn't a good way to scale it to a new value in the upgrade.  So he just gave it a new slot to force users to re-tune when they upgrade

Reply:
Yeah, this makes sense, it should be a new param key as has been done.